### DISCUSSION TITLE
Final comments on the mnt

### DISCUSSION THREAD

**Discussion by rodrigoSantamaria** [RebelusQuo/the-wall-is-down#22] [General]
I read the whole .mnt file, and have these comments. The first one may affect several cards, and you can ignore it, as I think we already gone through it and settle like it is in the mnt, but it still bugs me a little.

-1: I am not sure about the "Player gains", "add enough player influence". I mean, if you feel is necessary for making the card even clearer, do it, but it feels like super-redundant and very word consuming respect to "Gain" and "add enough influence". I understand that if nothing is said, the subject is the player executing the text. Most of the games are like that. 
-9: On a second thought, maybe "unplayable" is too strong and may give way to confusion.
If an East player plays it for ops, the text should trigger and EU will add 1 in Norway, even if in turns 6-8. It will not have the continuous effect, though. Maybe we can say *Continuous effect*: Norway is considered an Oil Production country. Cancel this effect at the beginning of turn 6. Ignore this effect if played on turns 6 to 8.
-12: on a second reading, do *Performed as an Event by the US player* may lead to 
confusion? I thought for a moment that it meant that only if the US performs it as an
Event, the text triggers, excluding when the EU performs it as an event as an effect of
East using the card for ops. I'm inclined to just remove that clause, I think the base rules
should be enough to understand how the wording works in this case.
-51: I decided to add "this additional VP loss cannot be prevented by Drones or War on Terror", to make the card a bit stronger.
-52: I decided to substitute Asia for Afghanistan to make the card a bit weaker and also to make it more accurate historically.
-56: Not sure if it's necessary the redundancy "US adds 1 US Influence in one EU country". Cannot it be just "US adds 1 Influence in one EU country"?

**Comment by RebelusQuo** [comment]
### "Player"

In most games a player performs the effects described on the cards they play themselves. However, here is a game where the effects can be performed by another player from the one playing the card, so there's cause for clear descriptions. Having said that, the term "player" is used in different contexts:

- "player gains N VP": This is meant to be consistent with "[power] gains N VP", the performing player rather than a specific player. However, it would work perfectly fine with "gain N VP". In fact, I see now that I've missed changing this on cards.
- "[add/remove] N player Influence": This is a real difference from simply "[add/remove] N Influence". The former restricts it to the performing players Influence only while the later allow the performing player to place any power's Influence. So ultimately the question is what you want the card effects to be.
- "each other player .. enough player Influence": This is related to the previous point, but here it's used as an opposite so as long as it's not meant to remove non-performing players Influence, then the performance is allowed to add *any player's* Influence then my argument is based on proper grammar.
- "The game ends after player loses 2 VP." On 'New World Order', the grammar is not very good with this one. It can definitely be "The game ends after losing 2 VP."
- "-1 VP per player’s Superpower country": Here we restrict the player's VP reduction to be relative to their own Superpower country, rather than any Superpower country that happen to be in the region. Unless I've actually misunderstood the concept, this should be needed.
- "the [US/Russian/Prevailing/active] player" / "[West/East] player": We've discussed this earlier, grammar.
- "Player looks at another player’s hand ..": Using "Look at another player's hand .." would be perfectly fine.

**Reply by RebelusQuo** [reply]
On second thought, we could just solve the "Influence" issue by general rule, to read as e.g. "place your own Influence if able to place at least one, otherwise *optionally* place Influence for any other player", or something similar. This way we can say "N Influence" and have a clear and precise definition of that what means.

**Reply by rodrigoSantamaria** [reply]
Ok, it was just a feeling it was redundant, but we can leave it as we decided earlier, no need to rethink it more. Playtest will give us more feedback.

**Reply by RebelusQuo** [reply]
So does this mean we stay with these definitions?

"N player Influence": the performing players kind of Influence.
"N Influence": any kind of Influence.

And if so, which cards should have specific, and which should have general? it's all up to you.

**Reply by rodrigoSantamaria** [reply]
Let's leave it as it is with these definitions, and the cards as are right now in the mnt.

**Comment by RebelusQuo** [comment]
### 9 EFTA agreement

> Add 1 EU Influence in Norway.
> **Played in turn 6-8**: Prevent this cards continuous effect.
> **Continuous effect**: Norway is considered an Oil Production country. Cancel this effect at the beginning of turn 6.

**Reply by rodrigoSantamaria** [reply]
Perfect

**Comment by RebelusQuo** [comment]
### 12 Empire of war

Ok, so here's the matrixes again:

With "Performed as an Event by the US player":
Played by an East player, US is next in HO: US performs the card effect.
Played by an East player, EU is next in HO: no effect.
Played by US as an Event: US performs the card effect.
Played by EU as an Event: no effect.

Without, and the instruction starts with "US conducts ..":
Played by an East player, US is next in HO: US performs the card effect.
Played by an East player, EU is next in HO: US performs the card effect.
Played by US as an Event: US performs the card effect.
Played by EU as an Event: US performs the card effect.

Without, and the instruction starts with "Player conducts ..", or "Conduct ..":
Played by an East player, US is next in HO: US performs the card effect.
Played by an East player, EU is next in HO: EU performs the card effect.
Played by US as an Event: US performs the card effect.
Played by EU as an Event: EU performs the card effect.

So basically, which behavior do you want?

**Reply by rodrigoSantamaria** [reply]
The second one, US perform the card effect always:
Without, and the instruction starts with "US conducts ..":
Played by an East player, US is next in HO: US performs the card effect.
Played by an East player, EU is next in HO: US performs the card effect.
Played by US as an Event: US performs the card effect.
Played by EU as an Event: US performs the card effect.

**Comment by RebelusQuo** [comment]
### 51 African Union

> Remove 1 West Influence from one African country except Morocco.
> **Permanent effect**: No more than 1 Influence can be added to African countries with a single Influence Ops. Destabilization Ops have an additional 1 VP loss. This additional VP loss cannot be prevented by the Drones Slot ability and 'War on Terror' Event effect.

**Reply by rodrigoSantamaria** [reply]
Perfect

**Comment by RebelusQuo** [comment]
### 52 Al Qaeda

> **Permanent effect**: After US conducts a Destabilization Ops in the Middle East, Africa, or Afghanistan the next East player by Header Order removes 1 West Influence from any country in the corresponding region.

**Reply by rodrigoSantamaria** [reply]
Perfect

**Comment by RebelusQuo** [comment]
### 56 Austerity plans

This ties in to both who makes the choice and restriction on the kind of Influence (or lack of).

"US adds N US Influence in X" would mean US chooses what X should be, but can only add their own Influence.
"US adds N Influence in X" would mean US chooses what X should be, and can add any kind of Influence.

"Please choose your weapon"

**Reply by rodrigoSantamaria** [reply]
Hehe. Yes, I know, but thinking on word economy and the most probable option, "US adds N influence in X" is shorter and 99.9% of the times a player will add its own influence. There's no reason to add any other player's influence except you do some kind of (unbinding) pacts with other players in exchange for future help.

**Reply by RebelusQuo** [reply]
Awaiting decision from the "Player" sub conversation.
